src/rpm/package.rs

```diff
@@ -83,13 +83,14 @@ impl RPMPackage {
         let digest_md5 = {
             use md5::Digest;
             let mut hasher = md5::Md5::default();
-            {
```

Comment:
I'm not 100% sure what this scope was for but it might have been to drop the buffer as soon as possible.